hazelcast/client.py

```diff
@@ -388,6 +389,10 @@ def __init__(self, **kwargs):
         self._invocation_service.init(
             self._internal_partition_service, self._connection_manager, self._listener_service
         )
+        self._internal_sql_service = _InternalSqlService(
+            self._connection_manager, self._serialization_service, self._invocation_service
+        )
+        self.sql = SqlService(self._internal_sql_service)
```

Comment:
The reasoning is that there should be some public methods on the SQL service so that they can be used within SqlResult. I could make them protected via `_` and hide them from users but this time, when I try to use them, IDE would show warnings for access to protected members which I would like to avoid. So, SqlService wraps InternalSqlService and exposes truly public APIs. We used a similar approach in a couple of other places.

So, I could merge them but this time, we should be okay with the IDE showing warnings

Reply:
> for access to protected members which I would like to avoid.

just wondering, what is protected members? functions with starts with _?

Reply:
yes